native/c/commands/ds.cpp

```diff
@@ -679,9 +697,12 @@ int handle_data_set_compress(InvocationContext &context)
   dds.push_back("alloc dd(sysprint) lrecl(80) recfm(f,b) blksize(80)");
   dds.push_back("alloc dd(sysin) lrecl(80) recfm(f,b) blksize(80)");
 
-  rc = zut_loop_dynalloc(dds);
+  ZDIAG diag = {};
```

Comment:
Per the changelog, should we disable the handler for compressing data sets?

Reply:
I dont think we supported compressing data sets in native Zowe Explorer so this is not a parity item.  I like leaving it as a non-functioning command in zowex hoping that someone will inquire about it if they try it.  Right now though, we don't have many exploiters of the `zowex` commands